wordfugue/sitesettings/middleware.py

```diff
@@ -3,9 +3,12 @@
 from .models import SiteSettings
 
 
-class SiteSettingsMiddleware(object):
+class SiteSettingsMiddleware:
```

Comment:
The way Django does middleware was changed in Django 2.1. So the underlying logic didn't change for this piece of middleware, but the interface did.